testsuite/testscripts/test-zeppelin.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import os
+import urllib2
+
+master_url = "http://" + os.environ['ZEPPELIN_MASTER_NODE'] + ":" + os.environ['default_zeppelin_port']
+
+response = urllib2.urlopen(master_url).getcode()
+
+if response == 200:
+    return 0
```

Comment:
hmmm, I think we'll need a print message indicating success?  So that the test can be validated in test-validate.sh later on?